project/plans_wip/PLAN-chat-assistant-ui-upgrade-v0.11.md

```
@@ -4,6 +4,16 @@
 
 Upgrade `@assistant-ui/*` packages in `apps/chat` to the latest `0.11.x` series (from `0.10.x`) to pick up upstream fixes/features, while keeping the current “external store runtime” architecture.
 
+## Progress (feat/chat-gpt-5-1)
+
+**Done on this branch**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)